app/helpers/document_type_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module DocumentTypeHelper
+  def t_doctype_exists?(edition, partial_i18n_key)
+    i18n_key = prefix_document_type(edition, partial_i18n_key)
+    I18n.exists?(i18n_key) || I18n.exists?(default_key(i18n_key))
+  end
+
+  def t_doctype_field(edition, partial_i18n_key)
```

Comment:
I think this would become a confusing abstraction, as we sometimes need to access a piece of guidance (`title.guidance.title`) and sometimes a label (`#{tag_field.id}.label`). A hybrid approach of I18n and prefixing seems a good way forward for now.

Reply:
Hmm...I think we'll see a stronger need for this son, since `#{field.id}` will become the norm, as part of the upcoming work on "flexible fields". On the other hand, I get that writing `, "title")` is somewhat different to writing `, "guidance.title")`, in that the interpretation of the argument is different. I don't think it's a blocker for this PR, but perhaps something to revisit as we stop the hard-coding for title and summary.